Operators/Types/user/Dom/util/FloatSelector_56426136-3c66-4246-89e6-737f8fc8f8c9.t3ui

```diff
@@ -0,0 +1,116 @@
+{
+  "Id": "56426136-3c66-4246-89e6-737f8fc8f8c9"/*FloatSelector*/,
+  "Description": "A simple way to select a float from a series of float inputs by index.\n\nIf \"wrap\" is selected, indexes that exceed the number of provided values will wrap (via modulo). This allows you to cycle through your values using something like [Counter], for example",
+  "InputUis": [
```

Comment:
functionally nothing is different, but it's more convenient if you already have a list of things. personally I feel like one day it would be nice to have MultiInputSlots accept collections of List or IList, but for now this has been useful for me :)